src/get-api-resources.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node --nolazy -r ts-node/register
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { $ } from 'zx'
+
+const main = async function (): Promise<void> {
+  $.verbose = false
+  const apiResourcesRaw: string = (await $`kubectl api-resources | grep ' true '`).stdout
+  const lines = apiResourcesRaw.split('\n')
+  const trimmedLines = lines
```

Comment:
I have no clue what is happening below. Please make a separate function that just perform parsing and provide appropriate unit tests.

Reply:
Not used. Will remove this

Reply:
All right. If it is not used then please remove it in PR

Reply:
What is is marked as resolved ?

Reply:
This has no impact and it was part of my effort to bring `otomi validate-templates` back online. It is fine